.github/workflows/iroha2-dev.yml

```diff
@@ -150,3 +150,51 @@ jobs:
         with:
           name: telemetry
           path: target/telemetry
+
+  sonarqube-defectdojo:
+    runs-on: ubuntu-latest
+    container:
+      image: hyperledger/iroha2-ci:nightly-2024-01-12
+    steps:
+      - uses: actions/checkout@v4
+      - name: Download clippy report artifact
+        uses: dawidd6/action-download-artifact@v3.1.4
+        with:
+          workflow: iroha2-dev-pr-static.yml
+          name: clippy.json
+          search_artifacts: true
```

Comment:
We consider some magic to searching for the latest uploaded artifact.

Reply:
This seems to work for me, AFAICT from the [documentation](https://github.com/dawidd6/action-download-artifact?tab=readme-ov-file#usage)